demo/implementation/functions/catchPokemon/catchPokemon.unit.test.ts

```diff
@@ -0,0 +1,78 @@
+import MockDate from 'mockdate';
+
+import { EventStoreEventsDetails } from '@castore/core';
+import { catchPokemonCommand } from '@castore/demo-blueprint';
+import { mockEventStore } from '@castore/test-tools';
+
+import { pokemonsEventStore } from '~/libs/eventStores/pokemons';
+import { trainersEventStore } from '~/libs/eventStores/trainers';
+
+describe('Commands - catchPokemon', () => {
+  const pikachuId = 'pikachu1';
```

Comment:
I'd rather define these mock variables outside of the describe.
To easily export them in the future for exemple.